README.md

```diff
@@ -4,8 +4,37 @@ Erlenmeyer is a Go Proxy used to parse multiple Open Source TimeSeries DataBase
 
 ![Erlenmeyer: Time Series query translator](./assets/logo.png)
 
+
+## Building erlenmeyer
+
 You can test and run locally erlenemeyer following those [building steps](./doc/BUILDING.md).
 
+## Configuration
+
+You can retrieve a `config.sample.yml` file, that can be re-used to configure erlenmeyer. 
+
+```sh
+cp config.sample.yml erlenmeyer.yaml
+```
+
+In this `config.yaml` you will retrieve the Warp10 backend endpoint to set `warp_endpoint`. Erlenmeyer will use this endpoint to resolve WarpScript generated queries. 
```

Comment:
`config.yaml` is confusing as we have specified `erlenmeyer.yaml` a few lines above and we are referring again a couple lines below as `/Path/to/erlenmeyer.yaml`

Reply:
Indead :)